src/classes/frUtil.cls

```diff
@@ -101,4 +81,64 @@ public class frUtil {
         }
     }
     
+    public static void logRelationshipError(Entity errObject, String recordId, Entity missingRelationship, String relationshipId) {
+        logRelationshipError(errObject, recordId, missingRelationship, relationshipId, null);
+    }
+    
+    public static void logRelationshipError(Entity errObject, String recordId, Entity missingRelationship, String relationshipId, String optionalError) {
+        String relationshipObject = getErrorObjectFromEnum(missingRelationship);
+        String error = 'Failed to find related record ' + 
+                 relationshipObject + ' with Funraise Id: ' + relationshipId;
```

Comment:
I think that the original versions logged `recordId` here instead of `relationshipId`